app/templates/frameworks/updates.html

```diff
@@ -107,24 +105,36 @@
     {% if framework.clarificationQuestionsOpen %}
       <form method="post">
         <div class="grid-row">
+
           <div class="column-two-thirds">
+            {{ summary.heading("Ask a clarification question") }}
+
+            <p>Ask a clarification question if you need a better understanding of:</p>
+            <ul class="list-bullet">
+              <li>the legal documents</li>
+              <li>your responsibilities as a supplier</li>
+            </ul>
+            <br />
+            <p>Find out about the <a href="https://www.gov.uk/guidance/digital-outcomes-and-specialists-suppliers-guide">application process, what you need to apply and what services are eligible</a> in the guidance.
+            <br /><br />
+              <a href="{{ url_for('external.help') }}">Get help if there's a problem with your account</a>
+            </p>
             <input type="hidden" name="csrf_token" value="{{ csrf_token() }}" />
               {%
                 with
                 large=true,
-                question = "Ask a clarification question",
                 name = clarification_question_name,
-                hint = "You should ask any questions you have about {} here. The Crown Commercial Service will not respond to you individually. (Maximum 5000 characters per question.)".format(framework.name),
+                hint = "(Maximum 5000 characters per question)",
```

Comment:
Minor quibble, but we could remove the parentheses from here?

Reply:
I'll check with Stefan.

Reply:
Stefan says remove 'em. Commit incoming